### PR TITLE
Doc-196-Canonical-UA-I-embedding-Ubuntu-MAAS

### PR DESCRIPTION
**Pull request by Siboney1**
Introduced UA and provided steps; notable parameters.

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/771#" title="Author self-approved">Siboney1</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @Siboney1. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: content/docs/06-integrations/00-ubuntu.md [hunk 1]
```diff
@@ -35,26 +58,124 @@ Ubuntu is a Linux distribution based on Debian and composed mostly of free and o
...
+
+## Modifying the Presets for MAAS Cloud
+
+1. Palette allows you to include the Ubuntu Advantage service in the **Profile Layers** section, when you create a new cluster profile. 
+
+
+2. Give the new Pack a **Name**, **Version number**, **Description**, **Type**, and **Tags** and click the **Next** button.
+
+
+3. Select **MAAS** as the Infrastructure provider and click the **Next** button.
```
Comment (resmaa) [inline content/docs/06-integrations/00-ubuntu.md:135]: Not just MAAS

**Review by resmaa — CHANGES_REQUESTED** [review]

**Review by truegilb — CHANGES_REQUESTED** [review]
Please work on the inline comments.

File: content/docs/06-integrations/00-ubuntu.md [hunk 2]
```diff
@@ -35,26 +58,124 @@ Ubuntu is a Linux distribution based on Debian and composed mostly of free and o
...
+
+<br />
+
+#### Personal Use
+
+- Free Personal Token for up to three machines with UAI.
+
+For more information see [Ubuntu Advantage for Infrastructure](https://ubuntu.com/advantage) site.
+
+## Modifying the Presets for MAAS Cloud
```
[the remark this answers is not in the document]
Comment (Siboney1) [reply content/docs/06-integrations/00-ubuntu.md:127]: Okay, removed - "for MAAS Cloud"

File: content/docs/06-integrations/00-ubuntu.md (hunk 1 above)
Comment (Siboney1) [reply content/docs/06-integrations/00-ubuntu.md:135]: Okay removed "MAAS" and added "your cloud provider".

**Review by resmaa — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/771#pullrequestreview-1037792241" title="Approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/771#" title="Author self-approved">Siboney1</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by truegilb — APPROVED** [review]
Looks good.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/771#pullrequestreview-1037792241" title="Approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/771#" title="Author self-approved">Siboney1</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/771#pullrequestreview-1038241175" title="Approved">truegilb</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**